src/react-compat.js

```diff
@@ -18,6 +18,8 @@ let childrenToArray;
 let renderWithOptions;
 let unmountComponentAtNode;
 let batchedUpdates;
+let PropTypes;
+let createClass;
```

Comment:
should we start importing createClass from here in the tests that use it? otherwise we'll have plenty of warnings